### PR TITLE
Addition of extra match label on client-egress-to-fqdns-google policy

### PR DESCRIPTION
**Pull request by llivingstone**
Fixes policy bug with self deployed CoreDNS installations - cillium-cli#313

### CODE REVIEW

File: connectivity/manifests/client-egress-to-fqdns-google.yaml [hunk 1]
```diff
@@ -30,3 +30,6 @@ spec:
     - matchLabels:
         k8s:io.kubernetes.pod.namespace: kube-system
         k8s:k8s-app: kube-dns
+    - matchLabels:
+        k8s:io.kubernetes.pod.namespace: kube-system
+        k8s:k8s-app: coredns
```
Comment (pchaigno) [inline connectivity/manifests/client-egress-to-fqdns-google.yaml:35]: Nit: missing new line.

Comment (llivingstone) [reply connectivity/manifests/client-egress-to-fqdns-google.yaml]: Hey @pchaigno - Was just fixing this! Should be sorted now, apologies.

**Review by pchaigno — APPROVED** [review]

**Comment by llivingstone** [comment]
@pchaigno - Do i have to do anything else for the CI to pass?

**Comment by michi-covalent** [comment]
hi  @llivingstone thanks for the PR!

- for aks failure, right now it doesn't work with PRs from forked repos.
- for flow validation failures in https://github.com/cilium/cilium-cli/pull/320/checks?check_run_id=2830032012 we are working on stabilizing it in https://github.com/cilium/cilium-cli/pull/319.

i'll go ahead and merge it 🚀

**Review by michi-covalent — APPROVED** [review]

**Pull request merged**